Codecademy/gamut
# feat: add MiniRewardStarsIcon

### Overview



Add reward stars icon from [streamline](https://www.streamlinehq.com/icons/streamline-bold/social-medias-rewards-rating/mood/reward-stars-3)



### PR Checklist

- [x] Related to designs: [product feature marketing spec](https://www.figma.com/design/TQ3lr66cVWpcOAYRxApWZV/Ready-for-dev---Product-Feature-Marketing?node-id=3-64960&m=dev)
- [x] Related to JIRA ticket: [DOT-313](https://skillsoftdev.atlassian.net/browse/DOT-313)

#### Testing Instructions



check the storybook preview and ensure `MiniRewardStarsIcon` shows up along with all the other mini icons






[DOT-313]: https://skillsoftdev.atlassian.net/browse/DOT-313?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
